packages/editor/src/components/post-locked-modal/style.scss

```diff
@@ -4,9 +4,6 @@
 
 .editor-post-locked-modal__buttons {
 	margin-top: $grid-unit-30;
```

Comment:
I wanted to note that this should be a `$grid-unit-15`, and I think we're only using `$grid-unit-30` because of margin collapse.

Reply:
Good thought. I'd love to actually see the more stack-based approach for contents inside here, so we don't have to specify this margin at all, [something like this perhaps](https://codepen.io/joen/pen/KKmbyrM).

Reply:
I agree, or we should switch to top margins for paragraphs.